### DISCUSSION TITLE
The bot in Python?

### DISCUSSION THREAD

**Discussion by Shayan-Mazahir** [TiagoMarinho/Soph#28] [Q&A] [answered]
I really like what you did, and after 100000 tries... I am unable to make in Python... Would be nice to see the bot in Python, or if i can get the API (cause i cant find the API as well)

**Comment by TiagoMarinho** [comment, marked as answer]
the stable diffusion webui api documentation is available at `https://127.0.0.1:7860/docs` by default when you're running it locally with `--api`, if I recall correctly

the calls to said API are made in `Soph/src/artificial intelligence/requestimage.js` ([here](https://github.com/TiagoMarinho/Soph/blob/master/src/artificial%20intelligence/requestimage.js#L83)) if you want to check how that works currently in Soph so you can do the same in your own bot (or if you want to "port" Soph to python for whatever reason)

I hope this has answered any questions you had. If I misunderstood anything or you have any further questions feel free to ask again
